api_tests/Get_a_product_by_ID.yaml

```diff
@@ -0,0 +1,33 @@
+# Test generated by RoostGPT for test credwallet-integration-artillery-updated using AI Type Open AI and AI Model gpt-4
+
+
+config:
+  target: "{{ $processEnvironment.API_HOST }}"
+  phases:
+    - duration: 60
+      arrivalRate: 5
+  plugins:
+    expect: {}
+  payload:
+    path: "Get_a_product_by_ID.csv"
+    fields:
+      - "productId"
+    skipHeader: true
+
+scenarios:
+  - name: "Get a product by ID"
+    flow:
+      - get:
+          url: "/products/{{ productId }}"
+          headers:
+            Authorization: "{{ $processEnvironment.AUTH_TOKEN }}"
+          expect:
+            - statusCode: 200
+            - hasProperty: "id"
+            - hasProperty: "name"
+            - hasProperty: "description"
+            - hasProperty: "price"
+            - equals:
+                id: "{{ productId }}"
+        log: "Sent a GET request to /products/{{ productId }} to get a product by its id"
+        think: 1
```

Comment:
wrong indentation of think

after fixing it 
⠋ * GET /products/10 

  ok statusCode 200 
  ok hasProperty id 
  ok hasProperty name 
  ok hasProperty description 
  ok hasProperty price 
  ok equals 10 
⠙ * GET /products/10 

  ok statusCode 200 
  ok hasProperty id 
  ok hasProperty name 
  ok hasProperty description 
  ok hasProperty price 
  ok equals 10 
⠹ * GET /products/10 

  ok statusCode 200 
  ok hasProperty id 
  ok hasProperty name 
  ok hasProperty description 
  ok hasProperty price 
  ok equals 10 
* GET /products/9 

  ok statusCode 200 
  ok hasProperty id 
  ok hasProperty name 
  ok hasProperty description 
  ok hasProperty price 
  ok equals 9 
⠸ * GET /products/8 

  ok statusCode 200 
  ok hasProperty id 
  ok hasProperty name 
  ok hasProperty description 
  ok hasProperty price 
  ok equals 8 
⠹ * GET /products/7 

  ok statusCode 200 
  ok hasProperty id 
  ok hasProperty name 
  ok hasProperty description 
  ok hasProperty price 
  ok equals 7 
⠸ * GET /products/8 

  ok statusCode 200 
  ok hasProperty id 
  ok hasProperty name 
  ok hasProperty description 
  ok hasProperty price 
  ok equals 8 
⠼ * GET /products/1 

  ok statusCode 200 
  ok hasProperty id 
  ok hasProperty name 
  ok hasProperty description 
  ok hasProperty price 
  ok equals 1 
* GET /products/9